### DISCUSSION TITLE
How to upgrade the theme? (The command in the document can‘t be upgraded)

### DISCUSSION THREAD

**Discussion by Ackerven** [vuepress-theme-hope/vuepress-theme-hope#2723] [Q&A] [answered]
I executed `pnpm add @vuepress/client@next vue vuepress@next vuepress-theme-hope -E && pnpm i && pnpm up
`, but the theme was not upgraded. I found this command here. [https://theme-hope.vuejs.press/zh/cookbook/tutorial/command.html](https://theme-hope.vuejs.press/zh/cookbook/tutorial/command.html#%E5%8D%87%E7%BA%A7%E7%89%88%E6%9C%AC)
![image](https://user-images.githubusercontent.com/70303900/215327961-2b633a1e-c5fb-41a6-ac61-5add206cccde.png)

**Comment by Mister-Hope** [comment]
You should add search-pro as well if you are using it

**Reply by Ackerven** [reply]
I tried adding `vuepress-plugin-search-pro` to it, but it didn't work.
![image](https://user-images.githubusercontent.com/70303900/215396923-a1a59894-4d70-420e-9ace-d6df82737932.png)
Can you give me a specific command?

**Reply by Ackerven** [reply]
Now the latest version of the `vuepress-theme-hope` is v2.0.0-beat.170, I want to upgrade the theme from 168 to 170.

**Reply by Mister-Hope** [reply]
Oh sorry, I will upgrade docs, all package should be marked with @latest

**Comment by Mister-Hope** [comment, marked as answer]
Docs have been updated.

- npm: `npx vp-update`
- yarn: `yarn dlx vp-udpate`
- pnpm: `pnpm dlx vp-update`
